deploy/crds/ApplicationMonitoring.yaml

```diff
@@ -39,4 +39,8 @@ spec:
               description: The list of namespaces to watch for prometheus custom resources
             alertmanagerInstanceNamespaces:
               type: string
-              description: The list of namespaces to watch for alertmanager custom resources
+              description: The list of namespaces to watch for alertmanager custom resources 
+            selfSignedCerts:
+              type: boolean
+              description: Indicates whether self signedcerts are being used in the cluster. Used by blackbox exporter
```

Comment:
remove empty line and tidy up wording on description. `self signedcerts` => `self signed certs`